lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -33,13 +34,16 @@ pub struct SparseVectorIndex<TInvertedIndex: InvertedIndex> {
 }
 
 impl<TInvertedIndex: InvertedIndex> SparseVectorIndex<TInvertedIndex> {
-    /// Create new sparse vector index
+    /// Open a sparse vector index at a given path
     pub fn open(
         id_tracker: Arc<AtomicRefCell<IdTrackerSS>>,
         vector_storage: Arc<AtomicRefCell<VectorStorageEnum>>,
         payload_index: Arc<AtomicRefCell<StructPayloadIndex>>,
         path: &Path,
     ) -> OperationResult<Self> {
+        // create directory if it does not exist
+        create_dir_all(path)?;
```

Comment:
I think the `SparseVectorIndex` is responsible for creating the index folder.

Reply:
Agreed